app/controllers/api/projects_controller.rb

```diff
@@ -67,9 +67,16 @@ def updated
 
   def dependencies
     subset = params.fetch(:subset, "default")
-    project_json = find_project_as_json_with_dependencies!(params[:platform], params[:name], params[:version], subset)
 
-    render json: project_json
+    if params[:v2] == "true"
+      @project = Project.find_best!(params[:platform], params[:name], [:repository, :versions])
```

Comment:
Not for now: This find_best! method is goofy and is a pile of spaghetti to track. Ultimately we index on 'lower((platform)::text), lower((name)::text)` so why not just skip to that. 🤷‍♂️